pkg/minikube/constants/constants_darwin.go

```diff
@@ -26,3 +26,4 @@ var SupportedVMDrivers = [...]string{
 
 const DefaultVMDriver = "xhyve"
 const OC_BINARY_NAME = "oc"
+const BINARY_NAME = "minishift"
```

Comment:
@dharmit , please remove this and other related changes as it is not being used now.